packages/react/README.md

```diff
@@ -139,6 +139,10 @@ function App() {
 }
 ```
 
+## useFetch
+
+When using Nile with multiple webservers, you can piggyback off of Nile to enforce access to your own backend. See the [useFetch](./src/lib/hooks/useFetch/README.md) documetation for more information
```

Comment:
1. Is "enforce access" accurate?  Or does it "enable access"?
2. Maybe add another line that elaborates on the scenario to clarify why `useFetch` is needed in the first place.

Reply:
1) it's whatever the developer wants to do, could be enforce, could be enable, could just be requesting.
2) the existing readme has more information on why you would use it. the idea here is to know if you want to read more about it. the customer use case is described ehre.